ci/install-aws.inc.sh

```diff
@@ -11,9 +11,21 @@ if [[ "$OS" = "linux" ]] && [[ "${FORCE_BREW:-}" != "true" ]]; then
     pip3 install --upgrade awscli
     echo_done
 else
-    echo_do "brew: Installing AWS utils..."
-    brew_install awscli
-    echo_done
+    if [[ ${GITHUB_ACTIONS:=} == "true" ]]; then
+        # https://github.com/actions/virtual-environments/commit/0fa2247a898625b8020d8dfb1dc4a54ddb2b256c
+        # Github Actions CI already installs awscli via pkg,
+        # which will make 'brew install awscli' fail when linking.
+        # See https://github.com/rokmoln/support-firecloud/runs/1291359454
+        pkgutil --pkgs | grep -q "^com\.amazon\.aws\.cli2" && HAS_AWSCLI_PKG=true || HAS_AWSCLI_PKG=false
+    fi
+    if [[ "${HAS_AWSCLI_PKG:=}" == "true" ]]; then
+        echo_skip "brew: Installing awscli... CI already has awscli preinstalled"
+    else
+        echo_do "brew: Installing AWS utils..."
+        brew_install awscli
+        echo_done
+    fi
+    unset HAS_AWSCLI_PKG
```

Comment:
I don't think this is needed since `HAS_AWSCLI_PKG` is only visible locally in this script. I would have guessed that we normally don't unset all local variables but not sure.

Reply:
Afaik this file is getting sourced in different other files, which would make the variable available in those files as well

Reply:
Ah, I did not think about the sourcing case.